### PR TITLE
Retarget keyboard strategies when grouplike element is selected

### PR DESCRIPTION
**Pull request by bkrmendy**
## Description
This PR upgrades our keyboard strategies so they work with grouplike/content-affecting elements. The way to accomplish this is that when a grouplike element is selected (such as a fragment), the keyboard strategies are retargeted to the children of the grouplike element using `retargetStrategyToChildrenOfContentAffectingElements`.

The strategies modified by this PR are:
- `keyboard-set-font-size`
- `keyboard-set-font-weight`
- `keyboard-set-font-opacity`
- `keyboard-absolute-move`
- `keyboard-absolute-resize`

The keyboard reorder strategy is tweaked as well, since it clashed with the keyboard resize strategy when a fragment was selected.

### CODE REVIEW

File: editor/src/components/canvas/canvas-strategies/strategies/keyboard-reorder-strategy.ts [hunk 1]
```diff
@@ -108,10 +109,11 @@ function fitness(interactionSession: InteractionSession | null): number {
...
   const accumulatedPresses = accumulatePresses(interactionSession.interactionData.keyStates)
-  const matches = accumulatedPresses.some((accumulatedPress) =>
-    Array.from(accumulatedPress.keysPressed).some(
-      (key) => key === 'left' || key === 'right' || key === 'up' || key === 'down',
-    ),
+  const matches = accumulatedPresses.some(
+    (accumulatedPress) =>
+      Array.from(accumulatedPress.keysPressed).some(
+        (key) => key === 'left' || key === 'right' || key === 'up' || key === 'down',
+      ) && shallowEqual(accumulatedPress.modifiers, emptyModifiers),
```
Comment (Rheeseyb) [inline editor/src/components/canvas/canvas-strategies/strategies/keyboard-reorder-strategy.ts:116]: We have an equality function for modifiers here: <https://github.com/concrete-utopia/utopia/blob/master/editor/src/utils/modifiers.ts#L89-L100>

**Comment by Rheeseyb** [comment]
I'm very unsure why this PR would have affected the render count tests...

File: editor/src/components/canvas/canvas-strategies/strategies/keyboard-reorder-strategy.ts (hunk 1 above)
Comment (bkrmendy) [reply editor/src/components/canvas/canvas-strategies/strategies/keyboard-reorder-strategy.ts]: thanks! fixed in [2c7b067](https://github.com/concrete-utopia/utopia/pull/3389/commits/2c7b067e2b06fe17a5d2d77624360f0e59a1e241)

**Comment by bkrmendy** [comment]
> I'm very unsure why this PR would have affected the render count tests...

It's the keyboard absolute resize strategy (found out through trial and error 😅), I assume the reason is that with this change, multiple elements are resized (the children of the grouplike container, not just the parent element), and that is causing the re-renders

**Review by seanparsons — APPROVED** [review]

**Review by Rheeseyb — APPROVED** [review]

**Pull request merged**